action.yml

```diff
@@ -12,7 +12,10 @@ inputs:
     required: false
     default: patch
   reviewers:
-    description: Reviewers to add to PR in case auto merge is not possible
+    description: Reviewers to add to PR in case auto merge is not possible. Comma separated list
+    required: false
+  team_reviewers:
```

Comment:
Do you need two params? Can't reviewers contain either a user or a team?

Reply:
That's what I thought, but the API exposed by GitHub actually separates the two, so they need to be provided separately

We could probably dump the values from reviewers into both reviewers and team_reviewers when calling the API, but it feels error-prone like this. Currently if there is no match, the API returns OK, but that might change at some point